package.json

```diff
@@ -12,148 +18,48 @@
     "node": ">=10.14.1"
   },
   "scripts": {
-    "build": "npx lerna run --stream build -- -s",
-    "clean:all": "npx lerna run clean && npx lerna clean --yes && rimraf ./node_modules",
-    "clean:dist": "npx lerna run clean",
+    "build": "yarn workspaces run build",
+    "build:dev": "yarn workspaces run build:dev",
+    "clean:all": "yarn workspaces run clean && rimraf ./node_modules",
```

Comment:
would we also want to clear the yarn [cache](https://yarnpkg.com/lang/en/docs/cli/cache/#toc-yarn-cache-clean)?

Reply:
I think I have more questions around when to use `yarn workspaces` and when to leverage our use of `lerna bootstrap` to install all of our package dependencies, and running commands like `lerna run clean` to remove all the package node_modules folders for us. Isn't lerna our monorepo package manager? 🤔

Reply:
A main objective of this change is to stop using `lerna` for most of our monorepo tasks—especially for install, build, and clean operations. The goal is to have each package describe its own dependencies **accurately**.

Run `yarn` from the root directory and it will install all dependencies for the monorepo. Similarly, `yarn build:dev` and `yarn:clean` from the root will use `workspaces` to run that command in each workspace. It should be faster.

Reply:
sounds good, i think from our discussion I'd like to remove lerna as a tool then if we are going to switch to using these workspaces and clear it out of our dependencies.